src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerTest.java

```diff
@@ -67,7 +68,7 @@ public void should_return_a_401_when_the_user_did_not_have_any_roles() {
             .body("status", equalTo(HttpStatus.UNAUTHORIZED.value()))
             .body("message", equalTo("User did not have sufficient permissions to perform this action"));
 
-        common.cleanUpTask(taskId);
+        common.cleanUpTask(taskId, Common.REASON_COMPLETED);
```

Comment:
should the Task be Completed in a Cancellation scenario?

Reply:
Here we are validating against history table which is the Camunda state 'completed'

Reply:
I thought it was `Deleted` when you showed me on Termination?

Reply:
sorry - ignore that.  This is a 401 scenario